sdk/python/feast/api/registry/rest/metrics.py

```diff
@@ -321,20 +319,43 @@ async def recently_visited(
             user = getattr(request.state, "user", None)
         if not user:
             user = "anonymous"
-        project_val = project or (server.store.project if server else None)
         key = f"recently_visited_{user}"
-        logger.info(
-            f"[/metrics/recently_visited] Project: {project_val}, Key: {key}, Object: {object_type}"
-        )
-        try:
-            visits_json = (
-                server.registry.get_project_metadata(project_val, key)
-                if server
-                else None
-            )
-            visits = json.loads(visits_json) if visits_json else []
-        except Exception:
-            visits = []
+        visits = []
+        if project:
```

Comment:
Earlier logic was not considering all projects while showing metrics. Fixed it by listing projects first and get the entries for all projects.